tests/unit/test_bruteforce.cpp

```diff
@@ -1242,3 +1242,22 @@ TEST_F(BruteForceTest, testCosine) {
     VecSimBatchIterator_Free(batchIterator);
     VecSimIndex_Free(index);
 }
+
+TEST_F(BruteForceTest, testSizeEstimation) {
+    size_t dim = 128;
+    size_t n = 100;
+
+    VecSimParams params{.algo = VecSimAlgo_BF,
+                        .bfParams = BFParams{.type = VecSimType_FLOAT32,
+                                             .dim = dim,
+                                             .metric = VecSimMetric_Cosine,
+                                             .initialCapacity = n}};
+    size_t test_estimation = sizeof(BruteForceIndex);
+
+    test_estimation += n * (sizeof(idType) + sizeof(labelType)); // labelToIdLookup
+    test_estimation += n * sizeof(idType);                       // idToVectorBlockMemberMapping
+
+    size_t estimation = VecSimIndex_EstimateSize(&params);
+    ASSERT_GE(estimation, test_estimation);
+    ASSERT_LE(estimation, test_estimation + 50);
```

Comment:
Can you explain what is the reason for that `estimation` is in `[test_estimation, test_estimation+50]` (and not equals to `test_estimation`, and how did you infer it?